chsdi/templates/htmlpopup/base.mako

```diff
@@ -53,7 +53,7 @@
 <body>
   <div class="chsdi-htmlpopup-container">
 % else:
-  <div class="htmlpopup-container">
+  <div id="${c['layerBodId']}#${c['featureId']}" class="htmlpopup-container">
```

Comment:
If not, I think it's a database problem, you shouldn't have 2 features with the same id in a layer.